src/cmakeWorkspaceFolder.ts

```diff
@@ -31,7 +31,7 @@ export interface DiagnosticsSettings {
     configureOnOpen: boolean | null;
 }
 
-export class CMakeToolsFolder {
+export class CMakeWorkspaceFolder {
```

Comment:
Did you like CMakeWorkspaceFolder over WorkspaceFolder?  I thought we were going to pick the shorter name since WorkspaceFolder isn't a CMake concept.